__tests__/app.test.js

```diff
@@ -366,16 +366,113 @@ describe('GET /api/reviews/:review_id', () => {
                 const { review } = body
                     expect(review).toMatchObject({
                         review_id: 2,
+                        title: 'Jenga',
+                        review_body: 'Fiddly fun for all the family',
+                        designer: 'Leslie Scott',
+                        review_img_url: 'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+                        votes: 5,
+                        category: 'dexterity',
+                        owner: 'philippaclaire9',
+                        created_at: expect.any(String),
+                        comment_count: expect.any(String)
+                    })
+        })
+    })
+})
+
+//11 GET api/reviews/<query>
+describe('GET /api/reviews/?category=<category>', () => {
+    test('status:200, responds with reviews by category', () => {
+        return request(app)
+            .get('/api/reviews?category=dexterity')
+            .expect(200)
+            .then(({body}) => {
+                const { reviews } = body
+                reviews.forEach((review) => {
+                    expect(review).toEqual({
+                        owner: expect.any(String),
                         title: expect.any(String),
-                        review_body: expect.any(String),
-                        designer: expect.any(String),
+                        review_id: expect.any(Number),
+                        category: expect.any(String),
                         review_img_url: expect.any(String),
+                        created_at: expect.any(String),
                         votes: expect.any(Number),
-                        category: expect.any(String),
+                        designer: expect.any(String),
+                        comment_count: expect.any(String)
+                    })
+                })
+            })
+    })
+    test('status:200, responds with reviews by category', () => {
+        return request(app)
+            .get('/api/reviews?category=dexterity')
+            .expect(200)
+            .then(({body}) => {
+                const { reviews } = body
+                reviews.forEach((review) => {
+                    expect(review.category).toBe('dexterity')
+                })
+            })
+    })
+    test('status:404, responds with error', () => {
+        return request(app)
+            .get('/api/reviews?category=notACategory')
+            .expect(404)
+            .then(({body}) => {
+                expect(body.msg).toBe('category not found')
+            })
```

Comment:
Another test could be useful. What if the category does exist but has no reviews associated with it? This should be a 200 with an empty array returned